draft-ietf-acme-acme.md

```diff
@@ -1944,11 +1932,13 @@ for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
```

Comment:
This seemed redundant to me as DNS only (currently) supports the ASCII character set and therefore seems to repeat guidance from numerous other DNS RFCs.